integration_tests/transaction_stress_test.go

```diff
@@ -0,0 +1,147 @@
+package integration_tests
+
+// package imports
+import (
+	"context"
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/peggyjv/gravity-bridge/module/v2/x/gravity/types"
+)
+
+// We have 4 validators running so this totals to 100 tx's
+const transactionsPerValidator int64 = 25
+const cosmosSentAmt int64 = 100
+const ethSentAmt int64 = 1
+
+func (s *IntegrationTestSuite) TestTransactionStress() {
+	s.Run("Transaction stress test", func() {
+		fmt.Println("StressTestTransaction starting")
+
+		// Approve spend & verify funds
+		for _, validator := range s.chain.validators {
+			err := s.SendEthTransaction(&validator.ethereumKey, testERC20contract, PackApproveERC20(gravityContract))
+			s.Require().NoError(err, "error approving spend")
+
+			balance, err := s.getEthTokenBalanceOf(common.HexToAddress(validator.ethereumKey.address), testERC20contract)
```

Comment:
`s.getEthTokenBalanceOf` will use the Geth client, right?

Reply:
yes